### PR TITLE
fix(@clayui/tabs): displayType should output nav-tabs variants

### LINKED ISSUE #5462: @clayui/tabs: The displayType property doesn't output anything and document nav-tabs-light
`displayType="light"` doesn't output the class `nav-tabs-light`.

We added a new variant in https://github.com/liferay/clay/pull/5461. We need to document it.

/cc @ethib137

### PR DESCRIPTION
**Pull request by pat270**
fixes #5462

### CODE REVIEW

File: packages/clay-tabs/docs/index.js [hunk 1]
```diff
@@ -63,6 +63,66 @@ const Tabs = () => {
...
+				</ClayTabs.TabPane>
+				<ClayTabs.TabPane aria-labelledby="tab-2">
+					2. Proin efficitur imperdiet dolor, a iaculis orci lacinia eu.
+				</ClayTabs.TabPane>
+				<ClayTabs.TabPane aria-labelledby="tab-3">
+					3. Proin efficitur imperdiet dolor, a iaculis orci lacinia eu.
+				</ClayTabs.TabPane>
+			</ClayTabs.Content>
+		</>
+	);
```
Comment (matuzalemsteles) [inline packages/clay-tabs/docs/index.js:109]: @pat270 We can update these examples to use the new compositing pattern now

File: packages/clay-tabs/src/List.tsx [hunk 2]
```diff
@@ -114,6 +114,7 @@ export const List = React.forwardRef<HTMLUListElement, IProps>(function List(
 			className={classNames(
 				'nav nav-tabs',
 				{'nav-justified': justified},
+				{[`nav-tabs-${displayType}`]: displayType},
```
Comment (matuzalemsteles) [inline packages/clay-tabs/src/List.tsx:117]: I think it's better to just add when the type is `light` to make sure that previous versions don't add the type to `basic` or `underline`.

```suggestion
				{[`nav-tabs-${displayType}`]: displayType === 'light'},
```

File: packages/clay-tabs/docs/index.js (hunk 1 above)
Comment (pat270) [reply packages/clay-tabs/docs/index.js:109]: @matuzalemsteles I'm not sure what you mean by new compositing pattern. Can you point me to an example?

Comment (matuzalemsteles) [reply packages/clay-tabs/docs/index.js:109]: You can see the example in the storybook and do the same here https://storybook.clayui.com/?path=/story/design-system-components-tabs--new-default

**Pull request merged**